.github/workflows/release-wc-beta-tester.yml

```diff
@@ -20,7 +20,8 @@ jobs:
             - name: Setup WooCommerce Monorepo
               uses: ./.github/actions/setup-woocommerce-monorepo
               with:
-                install: '@woocommerce/plugin-woocommerce-beta-tester'
+                install: '@woocommerce/plugin-woocommerce'
+                build: '@woocommerce/plugin-woocommerce'
```

Comment:
Do we need to build the woocommerce plugin for WooCommerce Beta Tester release? If not, I think we can remove the build command. 🤔

Reply:
@chihsuan To be honest, I don't fully understand why we need it...but it was needed to fix the errors. My best guess is that the packages in `packages/js` are built with `plugin-woocommerce`

Reply:
Interesting, sounds like improperly declared dependencies pnpm should handle that.

Reply:
@psealock I don't see anything obvious in https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-beta-tester/package.json

What do you think about merging it for now and following up to look into it further?

Reply:
Thanks @ObliviousHarmony for the explanation. I'll go ahead and merge this for now.